pkg/cmd/roachtest/tests/asyncpg.go

```diff
@@ -0,0 +1,114 @@
+// Copyright 2021 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package tests
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster"
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/registry"
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/spec"
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test"
+)
+
+func registerAsyncpg(r registry.Registry) {
```

Comment:
nit: to make this Go style compliant, we should call everything `AsyncPG` (acronyms are always the same case, capital letters if it is not the first word, e.g. pgCompat, compatPG)

Reply:
hmm, actually maybe ignore this -- it's stylised asyncpg /shrug